### ISSUE TITLE
For iOS: when tapping an RX.Button, VoiceOver doesn't announce the button's accessibilityLabel.

### ISSUE THREAD

**Issue by jessamp** [Microsoft/reactxp#810]
Not sure if this is related to the accessibility bug already mentioned, but I'm creating a button component with the accessibilityLabel defined and when I click the button, nothing is read. Is there something else I need to define? The problem is only in IOS.

**Comment by jessamp** [comment]
@mshoho any thoughts on this?
